programs/ssl/ssl_client2.c

```diff
@@ -801,26 +897,39 @@ int main( int argc, char *argv[] )
             goto usage;
     }
 
+    /* Event-driven IO is incompatible with the above custom
+     * receive and send functions, as the polling builds on
+     * refers to the underlying net_context. */
+    if( opt.event == 1 && opt.nbio != 1 )
+    {
+        mbedtls_printf( "Warning: event-driven IO mandates nbio=1"
+                        " - overwrite\n" );
+        opt.nbio = 1;
+    }
+
 #if defined(MBEDTLS_DEBUG_C)
     mbedtls_debug_set_threshold( opt.debug_level );
 #endif
 
     if( opt.force_ciphersuite[0] > 0 )
     {
         const mbedtls_ssl_ciphersuite_t *ciphersuite_info;
-        ciphersuite_info = mbedtls_ssl_ciphersuite_from_id( opt.force_ciphersuite[0] );
+        ciphersuite_info =
+            mbedtls_ssl_ciphersuite_from_id( opt.force_ciphersuite[0] );
 
         if( opt.max_version != -1 &&
             ciphersuite_info->min_minor_ver > opt.max_version )
         {
-            mbedtls_printf("forced ciphersuite not allowed with this protocol version\n");
+            mbedtls_printf( "forced ciphersuite not allowed with "
+                            "this protocol version\n" );
```

Comment:
Please don't break up string literals. People grep for them.

Reply:
@gilles-peskine-arm Honestly, I have done that in multiple places throughout other PR's, too, in order to adhere to our style requirements. Not sure what we want to give preference here.

Reply:
@hanno-arm Our coding standards state that 80 characters is the “preferred” maximum length, and if there ever was a good reason to break this general rule, it's to fit string literals on a single line. Even the extremely picky rules for the Linux kernels forbid breaking debug messages.

Reply:
I think historically we used to break string literals in order to fit in the 80 characters limit, but I agree with Gilles that we shouldn't. We might want to edit the style document to clarify that.

Reply:
@mpg @gilles-peskine-arm Given that there'll be many occasions where this needs to be changed, I'd prefer to keep this PR as is and to postpone documenting and implementing this style requirement to a separate PR. Would you be ok with that?

Reply:
@hanno-arm I don't expect this PR to fix any preexisting split debugging messages, but please don't introduce any new one.